simpleui-server/.gitignore

```diff
@@ -2,6 +2,7 @@
 dist/*
 test/dist/*
 node_modules/*
+package-lock.json
```

Comment:
any reason for this? npm (and friends) typically recommend adding this to source control

Reply:
good catch, can't remember why I did that. Just fixed it